xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/Registration.xml

```diff
@@ -421,6 +421,16 @@
       {{translation key="core.register.invalidUsername"/}}
     #elseif($reg == -11)
       {{translation key="core.register.mailSenderWronglyConfigured"/}}
+    #elseif($reg == -9)
+      #set ($passwordRules = $services.security.registration.passwordRules)
+      #set ($ruleDescriptions = [])
+      #foreach ($rule in $passwordRules)
+        #set ($discard = $ruleDescriptions.add($services.localization.render("security.registration.passwordRules.${rule}.description")))
+      #end
+      #set ($rules = $stringtool.join($ruleDescriptions, ','))
```

Comment:
Actually I guess that's the one I should escape.

Reply:
Well actually this is going to be removed as I'm getting rid of the check in XWiki#createUser.